File: 06_27/c_fortran_matlab_python/for_test.c
#include <stdio.h>
void main(){
    int i, j, mul;
    for(i=0;i<100000;i++){
        for(j=0;j<10000;j++){
            if(i==99999 && j==9999){
                printf("C: looping is over!!\n");
            }
        }
    }
}

File: 06_27/c_fortran_matlab_python/for_test.f90
program for_test
implicit none
    integer :: i, j
    do i=1, 100000
        do j=1, 10000
            if (i==100000 .and. j==10000) then
                write(*,*) 'FORTRAN: looping is over!!!'
            end if
        end do
    end do
end program for_test
